src/server/render-widget.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
```

Comment:
My opinion is that I would not see a huge advantage to switch to Preact for the widget only at this point. Choosing a rendering engine is something that should not differ on a feature/story level but be part of the whole architectural decision.

It would be highly confusing for future contributors to work on a mixed React/Preact code base.